demos/python_demos/text_detection_demo/text_detection_demo.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python3
+"""
+Copyright (c) 2019 Borys Kabakov
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
Please don't use Matplotlib. These demos are packaged with the OpenVINO toolkit, whose version of OpenCV is definitely compiled with GUI support, so it is just an extraneous dependency.